packages/bot-web-ui/src/stores/app-store.js

```diff
@@ -148,7 +148,7 @@ export default class AppStore {
             () => {
                 if (
                     (!client.is_logged_in && client.is_eu_country) ||
-                    client.has_maltainvest_account ||
+                    (client.is_eu && window.location.pathname === routes.bot) ||
                     isEuResidenceWithOnlyVRTC(client.active_accounts) ||
                     client.is_options_blocked
```

Comment:
We will be needing to move this to a reusable function instead. It is present in multiple lines of the file.